English.md

```diff
@@ -25,7 +25,8 @@ We decided to maintain a single version of the list, to make updating easier. Pl
 
 You can use this index to navigate directly to the relevant sections:
 - [Technology](README.md#tecnologia)
- - [Incubators, Accelerators and Hubs](README.md#incubatori-acceleratori-e-hubs)
+- [Coworking spaces](README.md#coworking)
+- [Incubators, Accelerators and Hubs](README.md#incubatori-acceleratori-e-hubs)
```

Comment:
I currently added coworking at the same level of incubators etcetera. This is arguable but we can discuss it in a dedicated issue, and change it eventually in case